### PR TITLE
libxkbcommon

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - this is ready for review!

File: recipes/libxkbcommon/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+export CFLAGS="${CFLAGS} -lxcb -lxcb-xkb"
+
+meson setup build \
+  --prefix=$PREFIX \
+  --libdir=$PREFIX/lib \
+  --includedir=${PREFIX}/include \
+  -Denable-wayland=false \
+  -Denable-docs=false
+ninja -C build install -v
```
Comment (ocefpaf) [inline recipes/libxkbcommon/build.sh]: Is there a `ninja` test or something similar?

Comment (scopatz) [reply recipes/libxkbcommon/build.sh]: Not yet is documented that I can figure out

**Comment by scopatz** [comment]
Thanks @ocefpaf!

**Pull request merged**